main.go

```diff
@@ -48,6 +55,15 @@ func main() {
 		lib.NamespaceCostsPage(w, bucket, wantJson, client)
 	})
 
+	for _, namespace := range namespaces {
```

Comment:
You've got access to dynamic routes since go 1.22. SO you don't need to loop over the namespaces. You can just pass the path directly.

https://go.dev/blog/routing-enhancements

eg.
```
http.HandleFunc("GET /posts/{id}", handlePost2)
```